apps/api/src/report/types/milestone-table-entry.ts

```diff
@@ -0,0 +1,120 @@
+import { STATUS } from '@ien/common';
+
+/**
+ * milestone table entry for report 9 and 10
+ */
+export class MilestoneTableEntry {
+  id!: string;
+
+  registration_date!: Date;
+
+  [STATUS.APPLIED_TO_NNAS]: Date | null;
```

Comment:
should these have `?` ?

Reply:
I need to understand precisely how typescript works, but I'd like to omit it if possible.